style/layout/_main.scss

```diff
@@ -0,0 +1,197 @@
+.main {
+    display: flex;
+    flex-direction: row;
+
+    &__heading {
+        font-family: $primary-font;
+        font-size: 5rem;
+        font-weight: 600;
+        margin: 4rem 2rem;
+    }
+
+    &__posts {
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.post {
+    display: flex;
+    flex-direction: row;
+
+    &__img {
+        display: block;
+        width: 30rem;
+        height: 20rem;
+        margin: 0 2rem 2rem 2rem;
+
+        &--1 {
+            background: url('../../img/fire1.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+
+        &--2 {
+            background: url('../../img/leaf-home.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+
+        &--3 {
+            background: url('../../img/meal.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+    }
+
+    &__textbox {
+        display: block;
+        margin: 2rem 0;
+    }
+
+    &__subheading {
+        color: $color-primary;
+        font-family: $primary-font;
+        font-size: $font-size-small;
+        text-transform: uppercase;
+        font-weight: 600;
+        margin: 2rem 0;
+    }
+
+    &__heading {
+        font-size: 3rem;
+        margin: 0;
+    }
+
+    &__paragraph {
+        margin: 2rem 0;
+        line-height: 2;
+    }
+
+    &__morebox {
+        position: relative;
+    }
+
+    &__more {
+        font-family: $primary-font;
+        color: $color-black;
+        font-size: $font-size-small;
+        font-weight: 600;
+        text-decoration: none;
+        text-transform: uppercase;
+        margin-left: 4rem;
+
+        &::before {
+            content: "+";
+            font-family: $primary-font;
+            font-size: $font-size-big;
+            color: $color-primary;
+            display: inline-block;
+            transform: rotate(-45deg);
+            position: absolute;
+            bottom: -2rem;
+            left: 0;
+        }
+    }
+}
+
+.mission {
+    margin: 2rem;
+    margin-top: 6.5rem;
+    max-width: 40%;
+    position: relative;
+
+    &__back-subheading {
+        font-family: $primary-font;
+        color: $color-white;
+        font-size: $font-size-xxl;
+        font-weight: 500;
+        text-transform: uppercase;
+        word-break: break-all;
+        line-height: 0.7;
+        max-width: 40rem;
+
+        &::before {
+            content: "01";
```

Comment:
Znów, nie pamiętam layoutu, ale nie widzę nigdzie 02, 03? To na prwno jest numerowanie czy może tekst (który np może być inny). Wyobrażaj sobie, że tekst który masz w środku może być przykładowy, albo generyczny, to znaczy zmieniać się jak podepnie się np. szablon pod CMSa. Oczywiście są projekty, które trzeba w 100% odwzorować zgodnie z projektem, musisz też wiedzieć do jakiego przeznaczenia jest dana htmlka. Też możemy o tym więcej porozmawiać, bo mogłaś tego dokładnie nie zrozumieć, ciężko to tak wyjaśnić, ale takie rzeczy są dość istotne przy tworzeniu kodu :)

Reply:
Nie ma nigdzie 02, 03. Nie wiem do końca czemu to służy. Musisz mi powiedzieć, bo nie wiem czy Cie rozumiem...

Reply:
To to też obgadajmy na spotkaniu